src/graphql/search/data/index.ts

```diff
@@ -58,45 +54,40 @@ const index = async (
   // Get the results from the DataLoader
   const dataloaderResults: CombinedDataResult[] = await Promise.all(
     // Construct the keys e.g. the URLs that should be loaded or fetched
-    endpoints.map(
-      async ({ endpoint, type, searchParam, queryFormatter, params, label, labelSingular }) => {
-        // Remove search parameter prefixes
-        q = queryFormatter ? q?.replace(queryFormatter, '') : q
-
-        const query = queryString.stringify({ [searchParam]: q, page, ...(params || {}) })
-
-        const key = `${endpoint}/?${query}`
-        const result = await loaders.data.load(key)
-
-        // If an error is thrown, delete the key from the cache and throw an error
-        if (result.status === 'rejected') {
-          loaders.data.clear(key)
-
-          return {
-            count: 0,
-            label,
-            type,
-            results: [],
-            reason: result.reason,
-            status: result.status,
-          }
-        }
+    endpoints.map(async ({ endpoint, type, searchParam, params, label, labelSingular }) => {
+      const query = queryString.stringify({ [searchParam]: q, page, ...(params || {}) })
 
-        // The Promise resolved, so return the valid Data
-        const { results = [], count } = result.value || {}
+      const key = `${endpoint}/?${query}`
+      const result = await loaders.data.load(key)
+
+      // If an error is thrown, delete the key from the cache and throw an error
+      if (result.status === 'rejected') {
+        loaders.data.clear(key)
 
         return {
-          count: count || 0,
-          label: count === 1 ? labelSingular : label,
+          count: 0,
+          label,
           type,
+          results: [],
+          reason: result.reason,
           status: result.status,
-          results:
-            results.length > 0 && limit // TODO: Add test to see if the correct number of results is returned
-              ? results.slice(0, limit).map((result: Object) => normalizeResults(result, type))
-              : [],
         }
-      },
-    ),
+      }
+
+      // The Promise resolved, so return the valid Data
+      const { results = [], count } = result.value || {}
+
+      return {
+        count: count || 0,
+        label: count === 1 ? labelSingular : label,
+        type,
+        status: result.status,
+        results:
+          results.length > 0 && limit // TODO: Add test to see if the correct number of results is returned
```

Comment:
Why not add that test to this PR?

Reply:
A bit out of scope for this ticket (didn't add the todo), but will create a test for it